plugins/arDominionB5Plugin/modules/actor/templates/_contextMenu.php

```diff
@@ -0,0 +1,41 @@
+<div class="sidebar-lowering">
+
+  <?php foreach ($lists as $list) { ?>
+
+    <?php $pager = $list['pager']; ?>
+
+    <section class="card sidebar-paginated-list mb-4"
+      data-total-pages="<?php echo $pager->getLastPage(); ?>"
+      data-url="<?php echo $list['dataUrl']; ?>">
+
+      <div class="card-header text-center p-2">
+        <h5 class="mb-0">
+          <?php echo __('%1% of', ['%1%' => $list['label']]); ?>
+          <span class="invisible float-end spinner">
+            <i class="fas fa-spinner fa-spin ms-2" aria-hidden="true"></i>
+            <span class="visually-hidden"><?php echo __('Loading ...'); ?></span>
+          </span>
+        </h5>
+      </div>
+
+      <ul class="list-group list-group-flush small">
```

Comment:
Aggregations are not small, not sure if we want to keep this ones that way too (or the other way around).

Reply:
Fixed.